docs/proposals/20230726-integrate-keda-http-add-on.md

```diff
@@ -0,0 +1,285 @@
+          - type: binding.kafka 
+            name: kafka-input
+          - type: pubsub.rocketmq
+            name: rocketmq-input
+            topic: sample
+          - type: state.redis
+            name: redis-input 
+      dapr:
+        - type: binding.kafka
+          name: kafka-receiver
+        - type: pubsub.rocketmq
+          name: rocketmq-server
+          topic: sample
+        - type: state.redis
+          name: redis-output 
+    outputs:
+      - name: notify
+        component: notification-manager
+        operation: "post"
+    bindings:
+      kafka-receiver:
+        type: bindings.kafka
+        version: v1
+        metadata:
+          - name: brokers
+            value: "kafka-server-kafka-brokers:9092"
+          - name: authRequired
+            value: "false"
+          - name: publishTopic
+            value: "logs"
+          - name: topics
+            value: "logs"
+          - name: consumerGroup
+            value: "logs-handler"
+      notification-manager:
+        type: bindings.http
+        version: v1
+        metadata:
+          - name: url
+            value: http://notification-manager-svc.kubesphere-monitoring-system.svc.cluster.local:19093/api/v2/alerts
+    scaleOptions:
+      keda:
+        scaledObject:
+          pollingInterval: 15
+          minReplicaCount: 0
+          maxReplicaCount: 10
+          cooldownPeriod: 60
+          advanced:
+            horizontalPodAutoscalerConfig:
+              behavior:
+                scaleDown:
+                  stabilizationWindowSeconds: 45
+                  policies:
+                  - type: Percent
+                    value: 50
+                    periodSeconds: 15
+                scaleUp:
+                  stabilizationWindowSeconds: 0
+        # httpScaledObject:
+        #   minReplicaCount: 0
+        #   maxReplicaCount: 10
+        #   targetPendingRequests: 100
+        #   cooldownPeriod: 60
+        ## The triggers definition for KEDA should be moved back to keda
+        triggers:
+          - type: kafka
+            metadata:
+              topic: logs
+              bootstrapServers: kafka-server-kafka-brokers.default.svc.cluster.local:9092
+              consumerGroup: logs-handler
+              lagThreshold: "20"
+    template:
+      containers:
+        - name: function # DO NOT change this
+          imagePullPolicy: IfNotPresent
+    hooks:
+      pre:
+        - pre-hook1
+        - pre-hook2
+      post:
+        - post-hook2
+        - post-hook1
+    tracing:
+      enabled: true
+      provider:
+        name: opentelemetry
+        exporter:
+          name: jaeger
+          endpoint: "http://localhost:14268/api/traces"
+      tags:
+        func: sample-binding
+        layer: faas
+      baggage:
+        key: opentelemetry
+        value: v1.23.0
+```
+
+### Example
+
+#### Step 1: Configure Dapr state store component & Ingress
+
+#### Step 2: Create Input Deployment & Service & HTTPScaledObject
```

Comment:
HTTPScaledObject shouldn't be created by users, it should be created by OpenFunction controller automatically